packages/utils/src/types/ticket.ts

```diff
@@ -112,19 +113,49 @@ export class Ticket {
   }
 
   getHash(): Hash {
-    return toEthSignedMessageHash(u8aToHex(serializeUnsigned({ ...this })))
+    return toEthSignedMessageHash(
+      u8aToHex(
+        serializeToU8a([
+          [this.counterparty.serialize(), Address.SIZE],
+          [this.challenge.serialize(), Address.SIZE],
+          [this.epoch.serialize(), UINT256.SIZE],
+          [this.index.serialize(), UINT256.SIZE],
+          [this.amount.serialize(), Balance.SIZE],
+          [this.winProb.serialize(), Hash.SIZE],
+          [this.channelIteration.serialize(), UINT256.SIZE]
+        ])
+      )
+    )
   }
 
   static get SIZE(): number {
-    return Address.SIZE + Hash.SIZE + UINT256.SIZE + UINT256.SIZE + Hash.SIZE + UINT256.SIZE + Signature.SIZE
+    return (
+      Address.SIZE +
+      Address.SIZE +
+      UINT256.SIZE +
+      UINT256.SIZE +
+      Balance.SIZE +
+      Hash.SIZE +
+      UINT256.SIZE +
+      Signature.SIZE
+    )
   }
 
-  getSigner(): PublicKey {
+  getEmbeddedFunds(): Balance {
```

Comment:
is `getEmbeddedFunds` used anywhere in our code?

Reply:
removed, now obsolete